src/syscall/syscall_libc.go

```diff
@@ -68,6 +69,86 @@ func Rmdir(path string) (err error) {
 	return
 }
 
+type Timespec struct {
+	Sec  int64
+	Nsec int64
+}
+
+// Go chose Linux's field names for Stat_t, see https://github.com/golang/go/issues/31735
+type Stat_t struct {
```

Comment:
Same here. This should probably live in syscall_libc_darwin.go.

Reply:
Sorted.